kfdefs/overlays/moc/smaug/opf-jupyterhub/blackbox-exporter.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: blackbox-exporter
+spec:
+  port:
+    targetPort: 9115-tcp
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  to:
+    kind: Service
+    name: blackbox-exporter
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: blackbox-exporter
+    prometheus.io/scrape: "true"
+    prometheus.io/joblabel: "blackbox-metrics"
+  name: blackbox-exporter
+spec:
+  ports:
+  - name: 9115-tcp
+    port: 9115
+    protocol: TCP
+    targetPort: 9115
+  selector:
+    deployment: blackbox-exporter
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: blackbox
```

Comment:
change the name to `blackbox-exporter-config` to be more descriptive since there are other apps deployed in the same namespace

Reply:
done